README.md

```diff
@@ -1,2 +1,149 @@
-# Conference-Page
-An Online Website for a Conference.
+## Conference Page
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+
+- [🚀 Live Demo](#live-demo)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 <a name="about-project">Content Creators Global Summit</a>
+
+
+This repository contains the source codes for a Conference Page built for Content creators. It showcases my skills and expertise in building responsive mobile and desktop versions with HTML, CSS and Javascript. 
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.css3.com/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Home Page: An organized webpage containing major information about the event.**
+- **About Page:A linked webpage to Home Page giving more information about the event**
+- **Functionality: The mobile navbar, featured Speaker section were made easy for users to navigation the webpage with Javascript**
+- **Social Contact: A form allowing visitors to get in touch with me.**
+- **Responsive Design: The website is optimized for different devices and screen sizes.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link]()
```

Comment:
- [ ] Your live demo currently doesn't show the live working version of your project hosted on github pages. Please i kindly suggest your host your project using github pages. The live version is important as it demo's your project for anyone checking it out :+1: